src/Backends/Cubics/UNIFAC.h

```diff
@@ -20,7 +20,7 @@ namespace UNIFAC
     private:
         CoolProp::CachedElement _T; ///< The cached temperature
 
-        std::size_t N = 0; ///< Number of components
+        std::size_t N; ///< Number of components
```

Comment:
`set_components` should always be called, so there is no need to initialise before.
I don't know why I did initialised it to 0.

However, If we plan to use `add_component` alone somewhere to add components after having set them (add_component is public), this may be required to add something in `add_component` that would adapt `N`.